hieradata/class/vagrant/elasticsearch.yaml

```diff
@@ -0,0 +1,26 @@
+---
+
+govuk_elasticsearch::version: '1.7.5'
+govuk_elasticsearch::cluster_hosts: ['elasticsearch-1.backend:9300']
+govuk_elasticsearch::minimum_master_nodes: 1
+govuk_elasticsearch::aws_access_key: 'ABCDEFGHIJKLMNOPQRST'
+govuk_elasticsearch::aws_secret_key: 'n4AQffsnBJ5rcQK9DBn5eYHVuDqhT5DVCYa7ZU9tt'
+
+lv:
+  data:
+    pv: '/dev/sdb1'
+    vg: 'elasticsearch'
+
+mount:
+  /mnt/elasticsearch:
+    disk: '/dev/mapper/elasticsearch-data'
+    govuk_lvm: 'data'
+    mountoptions: 'defaults'
+    #FIXME: Adjust these values when elasticsearch is upgraded to 1.4
+    percent_threshold_warning: 31
+    percent_threshold_critical: 16
+
+govuk_elasticsearch::snapshot::indices:
+   '.kibana':
+     s3_bucket: 'jonauman-test2'
```

Comment:
Oops. Yes, that shouldn't be there. Also note that kibana 3 uses 'kibana-int' for the index whereas kibana4 uses '.kibana'. Do we want to assume 3 or 4 for vagrant purposes?